### PR TITLE
TDP-3620: Add transferDisconnect event

### PR DESCRIPTION
**Pull request by n-miles**

### CODE REVIEW

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-tdp-3620.s3-website-us-east-1.amazonaws.com/

File: voice/bxml/about.md [hunk 1]
```diff
@@ -49,16 +49,17 @@ BXML callbacks are HTTP messages that are sent to your application server to not
...
+| [Record Complete](callbacks/recordComplete.md)     | Bandwidth API sends this when a `<Record>` is completed. The recording is not available to download until the [Recording Available](callbacks/recordingAvailable.md) event is received.|
+| [Redirect](callbacks/redirect.md)                  | Bandwidth API sends this when a `<Redirect>` is requested or when a POST request is made to the [/calls/{callId}](../methods/calls/postCallsCallId.md) endpoint requesting a redirect. |
+| [Transfer Answer](callbacks/transferAnswer.md)     | Bandwidth API sends this when a `<Transfer>` is answered.                                                                                                                              |
+| [Transfer Complete](callbacks/transferComplete.md) | Bandwidth API sends this when the `<Transfer>` is complete.                                                                                                                            |
```
Comment (muesono-daitan) [inline voice/bxml/about.md]: \+ ` and the original call needs to continue.`
to match the description in the other page.

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-tdp-3620.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-tdp-3620.s3-website-us-east-1.amazonaws.com/

File: voice/bxml/callbacks/transferComplete.md [hunk 2]
```diff
@@ -1,7 +1,8 @@
 {% method %}
 ##  Transfer Complete Event – <Transfer> verb
-This event is sent to the callbackUrl of the Leg-A call when the transferred call(B-leg) completes.
-In a simultaneous ringing scenario, only one call leg-B succeeds and this event corresponds to that successful leg. If none of the calls were answered, transferComplete is not sent.
+This event is sent to the `transferCompleteUrl` of the A-leg's `<Transfer>` verb when the transferred call (B-leg) completes.
+In a simultaneous ringing scenario, only one B-leg succeeds and this event corresponds to that successful leg.
+If none of the calls were answered, the `transferComplete` event corresponds to one of the legs.
```
Comment (muesono-daitan) [inline voice/bxml/callbacks/transferComplete.md:5]: Could you please also fix the example below? (line ~53)
It says `"cause"            : "completed",` but that is incorrect. The cause should be `hangup`.

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-tdp-3620.s3-website-us-east-1.amazonaws.com/

**Review by muesono-daitan — APPROVED** [review]

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-tdp-3620.s3-website-us-east-1.amazonaws.com/

**Pull request merged**